### DISCUSSION TITLE
Diskusi seputar ide fitur yang kalian punya

### DISCUSSION THREAD

**Discussion by deaafrizal** [deaafrizal/cuyuniverse-forum#69] [Ideas]
**Kategori IDEAS Untuk Diskusi Seputar ide-ide liar yang kalian miliki**

_please don't be afraid to discuss something here 👇_

**Comment by novalramdhani** [comment]
Ijin menambahkan kang, tambahkan fitur untuk setiap postingan mempunyai tags dan kategorinya masing masing.

**Reply by deaafrizal** [reply]
setuju

**Comment by hyuzea** [comment]
portofolio / projects

**Reply by deaafrizal** [reply]
ini juga setuju

**Comment by hyuzea** [comment]
sosial media links

**Comment by wahmulyadi** [comment]
bisa mungkin next update tambahin notifikasi kalo ada yang like atau komen gitu bang.

**Reply by deaafrizal** [reply]
done sepertinya sama kawan kita yg ada disini

**Comment by bagussatoto** [comment]
Izin menambahkan fitur pada cumputing plug-in bang

**Reply by hyuzea** [reply]
kaya gimana tuh

**Reply by bayupriyambada** [reply]
keknya kalau ini kejauhan bre

**Reply by bagussatoto** [reply]
@bayupriyambada oke deh. 
saya coba cari yang lain.

**Comment by mdaushi** [comment]
saran buat to do list bang. fitur on process apa2 saja dll

**Comment by zulfio** [comment]
Tambah WYSIWYG di halaman buat postingan bang

**Reply by deaafrizal** [reply]
jangan dulu euy karna bukan blog soalnya, biar kek twitter aja minimal 😁
but thanks ide nya bro mngkin next time ada menu khusus blog

**Comment by zulfio** [comment]
Tambah fitur report bang:
1. report postingan
2. report akun

Alasan reportnya kaya:
1. spam
2. sara
3. mengandung konten dewasa
4. dll

**Reply by deaafrizal** [reply]
beuhh sedappp

**Reply by zulfio** [reply]
baru ngeh, berarti harus ada dulu admin panel/dashboard nya ya
